routes/api.php

```diff
@@ -51,6 +51,11 @@
     Route::put('stand/assignment', 'StandController@createStandAssignment');
     Route::delete('stand/assignment/{callsign}', 'StandController@deleteStandAssignment')
         ->where('callsign', VatsimCallsign::CALLSIGN_REGEX);
+
+    // Notifications
+    Route::get('notifications', 'NotificationController@getActiveNotifications');
+    Route::get('notifications/unread', 'NotificationController@getUnreadNotifications');
+    Route::put('notifications/read/{notification}', 'NotificationController@readNotification');
```

Comment:
I would probably make this parameter `{id}` as that's what would actually be passed?

Reply:
Could also add regex matching to the parameter to make sure its an integer

Reply:
b15036a

Reply:
2dfc065